langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -74,8 +123,18 @@ public isolated function round(decimal x, int fractionDigits = 0) returns decima
     name: "round"
 } external;
 
-# IEEE quantize operation.
-
+# Return a decimal with a specified value and exponent.
+# Return a decimal value that has the same value (except for rounding) as the first
+# argument, and the same exponent as the second argument.
+# This is the IEEE quantize operation.
+#
+# ```ballerina
+# decimal d = 4.16;
+# d.quantize(3.512) ⇒ 4.160
+# 
+# decimal:quantize(3.15, 3.512) ⇒ 3.150
```

Comment:
Shall we use a different value for `y` in the second example? May make sense to leave the first as is in both.

Reply:
`y` is still the same in all?

I was thinking two examples like

```ballerina
decimal:quantize(4.1624, 3.512) ⇒ 4.162
decimal:quantize(4.1624, 3.51) ⇒ 4.16
```

Reply:
Updated. Please check